tests/ErrorHandlerTest.php

```diff
@@ -48,12 +48,13 @@ public function testDefaultContentIsEmpty()
 
     public function testCheckExceptionAndContent()
     {
-        $exception = new Exception('ERMAGHERD');$line = __LINE__;
+        $exception = new Exception('ERMAGHERD');
+        $line = __LINE__;
```

Comment:
The line was important in the text content.

As the tests are currently failing, would you mind replace this line with 
```php
$line = __LINE__ - 1;
```

Reply:
Done